### ISSUE TITLE
Unable to install the provided deb package on bookworm-based raspbian 

### ISSUE THREAD

**Issue by tgurr** [badaix/snapcast#1163]
**Describe the bug**
Debian/Raspbian bookworm ships with libflac12 (https://packages.debian.org/bookworm/libflac12) instead of libflac8 (https://packages.debian.org/bullseye/libflac8) resulting in being unable to install the provided `snapclient_0.27.0-1_without-pulse_armhf.deb`.

```
pi@raspberrypi:~ $ sudo dpkg -i snapclient_0.27.0-1_without-pulse_armhf.deb
Selecting previously unselected package snapclient.
(Reading database ... 62433 files and directories currently installed.)
Preparing to unpack snapclient_0.27.0-1_without-pulse_armhf.deb ...
Unpacking snapclient (0.27.0-1) ...
dpkg: dependency problems prevent configuration of snapclient:
snapclient depends on libflac8 (>= 1.3.0); however:
 Package libflac8 is not installed.
```

Issuing `sudo apt-get -f install` like explained in the documentation:
https://github.com/badaix/snapcast/blob/develop/doc/install.md#using-dpkg
then just removes snapclient.

Upstream provided snapclient (apt install snapclient) on bookworm only ships the older 0.26.0.

**Steps to Reproduce**
1. `sudo dpkg -i snapclient_0.27.0-1_without-pulse_armhf.deb`

**Environment details**
- OS: Raspbian (bookworm)
- Snapcast version: 0.27.0
- Installed from a package

**Attach logfile if applicable**
-

**Comment by chiefy** [comment]
I believe I ran into the same issue, forked the repo and fixed some of the failing CI/CD, [check out the built packages here](https://github.com/chiefy/snapcast/releases/tag/v0.27.0) (x86_64 only) which then worked. This was back in July, so YMMV.

**Comment by audas** [comment]
> I believe I ran into the same issue, forked the repo and fixed some of the failing CI/CD, [check out the built packages here](https://github.com/chiefy/snapcast/releases/tag/v0.27.0) (x86_64 only) which then worked. This was back in July, so YMMV.

Will these work on Raspberry with 64bit operating system then?

It appears that snapserver is failing on the lastest  Pi Zero 2, wondering if 64Bit might help.

**Comment by seppeel** [comment]
These packages don't work on Raspberry Pi (armhf instead of x86_64)

**Comment by badaix** [comment]
After my vacation I will create a docker image from the raspian bookworm image and add it to the CI. 
Two hours more build time for my poor memory limited RPi4... :)

**Comment by audas** [comment]
Raspberry snapclient_0.27.0-1_without-pulse_armhf.deb
snapc
dpkg: dependency problems prevent configuration of snapclient:
 snapclient depends on libflac8 (>= 1.3.0); however:
  Package libflac8 is not installed.
 snapclient depends on libvorbisidec1 (>= 1.2.1+git20180316); however:
  Package libvorbisidec1 is not installed.

**Comment by badaix** [comment]
@chiefy don't know what you had to fix, but the `Package` CI creates Debian packages for buster, bullseye and bookworm for X64 and for ARM.
You can download them here: https://github.com/badaix/snapcast/actions/runs/6037321427
Packages for both architectures are built in Debian buster/bullseye/bookworm docker containers. 
Only the bookworm ARM version is missing because it's using raspbian, which was not available, but has been released a few days ago.
What you can find on the release page are buster images. Seems that I should provide packages for all Debian versions.

**Comment by chiefy** [comment]
> @chiefy don't know what you had to fix, but the `Package` CI creates Debian packages for buster, bullseye and bookworm for X64 and for ARM. You can download them here: https://github.com/badaix/snapcast/actions/runs/6037321427 Packages for both architectures are built in Debian buster/bullseye/bookworm docker containers. Only the bookworm ARM version is missing because it's using raspbian, which was not available, but has been released a few days ago. What you can find on the release page are buster images. Seems that I should provide packages for all Debian versions.

Yeah, it's been a few months, but I noticed that the DEB builds haven't run for a few months. In that time I think the base image changed in some way that breaks if you're trying to install the `.deb` on a more recent docker image of debian. Again, I can't recall what the exact issue was. Anyhow maybe we could add a timer to re-build the release every few months w/ time-date stamp? If I have time, I will try to reproduce. Thanks!

**Comment by badaix** [comment]
@tgurr I've created a Raspbian [bookworm docker container](https://hub.docker.com/repository/docker/badaix/raspios-bookworm-armhf-lite/general) for ARM and added it to the package CI: 40f3495
You can find a debian package for bookworm [snapcast_armhf-debian-bookworm](https://github.com/badaix/snapcast/suites/17363015632/artifacts/993043422), based on the develop branch here: https://github.com/badaix/snapcast/actions/runs/6561332062

The package is not tested, because I don't have a bookworm based ARM system at hand.
Please check if it is working for you, so I can add it the the upcoming releases.

**Comment by seppeel** [comment]
I tested on a Raspberry Pi 4 with latest Raspberry Pi OS (64 Bit - bookworm) installed.
Was able to install, start and stream music.
-> Looks good so far.

**Comment by tgurr** [comment]
> @tgurr I've created a Raspbian [bookworm docker container]
> Please check if it is working for you, so I can add it the the upcoming releases.

@badaix Thanks! Tested `snapclient_0.27.0-1_without-pulse_armhf.deb` which works fine for me here as well:

```
Okt 20 13:54:24 cube snapserver[282]: Hello from b8:27:eb:32:fd:c2, host: hifiberry1, v0.27.0, ClientName: Snapclient, OS: Raspbian GNU/Linux 12 (bookworm), Arch: armv7l, Protocol version: 2
Okt 20 13:54:26 cube snapserver[282]: Hello from b8:27:eb:2a:ff:03, host: hifiberry2, v0.27.0, ClientName: Snapclient, OS: Raspbian GNU/Linux 12 (bookworm), Arch: armv7l, Protocol version: 2
```

**Comment by yellowonblack** [comment]
Installed and all good so far on Pi Zero W.

ClientName: Snapclient, OS: Raspbian GNU/Linux 12 (bookworm), Arch: armv6l, Protocol version: 2

Thanks.

**Comment by pcwii** [comment]
attempting to get things working on an i3 unit running debian bookworm and I have this issue. Is there a fix or do we have to await the next release?

**Comment by badaix** [comment]
There are already bookworm packages available for the develop branch:

https://github.com/badaix/snapcast/actions/runs/7553442434

**Comment by badaix** [comment]
I think you must be logged in to be able to download build artifacts

**Comment by badaix** [comment]
Fixed in version [0.28.0](https://github.com/badaix/snapcast/releases/tag/v0.28.0)
